aspen/http/mapping.py

```diff
@@ -17,36 +17,42 @@ class Mapping(dict):
     clobbers to the list, while subscript access returns the last item. Think
     about it.
 
+    WARNING: this isn't thread-safe
```

Comment:
We're okay with this because it's not designed to be used across thread boundaries, right? Request instances will exist  in at most one thread?